config/checkstyle/suppressions.xml

```diff
@@ -5,5 +5,6 @@
         "https://checkstyle.org/dtds/suppressions_1_2.dtd">
 
 <suppressions>
-
+    <suppress checks="MethodName" files="com[\\/]wanderersoftherift[\\/]wotr[\\/]mixin[\\/]"/>
```

Comment:
I assume you are happy with this approach?

Reply:
Yeah. Looks perfect, only location where we allow it is in Mixins.

Though Mojang uses it in some other places too, so sometimes it is a hassle when we copy code :D